packages/api/src/workers/indexers/predictionMarketIndexer.ts

```diff
@@ -375,35 +468,26 @@ class PredictionMarketIndexer implements IResourcePriceIndexer {
         );
       }
 
-      // Create Parlay if not exists
-      const existingParlay = await prisma.parlay.findFirst({
-        where: {
+      // Create Parlay 
+      await prisma.parlay.create({
+        data: {
+          chainId: this.chainId,
+          marketAddress: log.address.toLowerCase(),
+          maker: eventData.maker.toLowerCase(),
+          taker: eventData.taker.toLowerCase(),
           makerNftTokenId: eventData.makerNftTokenId,
           takerNftTokenId: eventData.takerNftTokenId,
+          totalCollateral: eventData.totalCollateral,
+          refCode: eventData.refCode,
+          status: 'active',
+          makerWon: null,
+          mintedAt: Number(block.timestamp),
+          settledAt: null,
+          endsAt: endsAt ?? null,
+          predictedOutcomes: predictedOutcomes as unknown as object,
```

Comment:
as Record<string, any> 

or create a new interface for this

my 2c, can leave as is